Enemy/src/main/java/dk/sdu/se/f23/InVasion/enemy/Enemy.java

```diff
@@ -0,0 +1,15 @@
+package dk.sdu.se.f23.InVasion.enemy;
+
+import data.Entity;
+import data.Point;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Enemy extends Entity {
+    private List<Point> route = new ArrayList<>();
+
```

Comment:
maybe add a empty constructor, even though it isn't needed it would be nice for similarity